src/candle_components/candle_datetime.rs

```diff
@@ -0,0 +1,33 @@
+#![cfg(feature = "chrono")]
+use chrono::{DateTime, Utc};
+
+#[derive(Debug, Clone)]
+#[cfg_attr(feature = "serde", derive(serde::Serialize, serde::Deserialize))]
+pub struct CandleDateTime {
```

Comment:
I think it makes sense to call this `OpeningDateTime` or `OpenDateTime` or similar. Just drop the `Candle` as its implied here. In principle there could also by `CandleComponent`'s that note the time of the high or low price now that I think about it. So `OpenDateTime` seems to be a sensible name. Also remember to rename the filename.

Reply:
Great point, going with `OpenDateTime`.  :+1: